.github/workflows/integration_test.yaml

```diff
@@ -51,7 +51,7 @@ jobs:
       charmcraft-channel: ${{ needs.charmcraft-channel.outputs.channel }}
       extra-arguments: >-
         ${{needs.extra-args.outputs.args}} -k test_${{ matrix.suite }}
-        ${{ matrix.arch.id == 'arm64' && ' --lxd-containers --series=focal' || '' }}
+        ${{ matrix.arch.id == 'arm64' && ' --lxd-containers --series=jammy' || '' }}
```

Comment:
Remind me again what we're doing here?  Why did this need to switch to jammy?  Is this so we run everything on jammy when we test?

Reply:
Yes, I changed this to ensure that we deploy all the tests on jammy.